next.config.js

```diff
@@ -38,18 +37,22 @@ module.exports = withPlugins([
   withGraphql,
   withBundleAnalyzer,
   {
+    serverRuntimeConfig: {
+      NEXTAUTH_URL: siteUrl,
+    },
     env: {
       JWT_SECRET: process.env.JWT_SECRET || 'development-key',
       API_URL: process.env.API_URL || 'https://api.stage.mpdx.org/graphql',
       REST_API_URL:
         process.env.REST_API_URL || 'https://api.stage.mpdx.org/api/v2/',
-      SITE_URL: SiteUrl,
+      SITE_URL: siteUrl,
       CLIENT_ID: process.env.CLIENT_ID || '4027334344069527005',
       CLIENT_SECRET: process.env.CLIENT_SECRET,
       BEACON_TOKEN:
         process.env.BEACON_TOKEN || '01b4f5f0-7fff-492a-b5ec-d536f3657d10',
     },
     // Force .page prefix on page files (ex. index.page.tsx) so generated files can be included in /pages directory without Next.js throwing build errors
     pageExtensions: ['page.tsx', 'page.ts', 'page.jsx', 'page.js'],
+    productionBrowserSourceMaps: true,
```

Comment:
Was this needed to get these changes to work or was it just something that should have been set to ```true``` originally?

Reply:
No I just wanted to debug on prod and couldn't. Unrelated to this PR but thought would be nice to enable.